code/game/objects/structures/window.dm

```diff
@@ -490,3 +490,27 @@
 
 /obj/structure/window/shuttle/tinted
 	opacity = TRUE
+
+/obj/structure/window/reinforced/clockwork
+	name = "ratvarian window"
+	desc = "A paper-thin pane of translucent yet reinforced brass."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "clockwork_window_single"
+	debris = list(/obj/item/clockwork/component/vanguard_cogwheel)
```

Comment:
This will not work; if you want custom debris, you need to delete the debris spawned in New()'s ..() and add this to the debris.